packages/flutter/test/material/chip_test.dart

```diff
@@ -130,6 +132,64 @@ Future<void> _testConstrainedLabel(
   expect(chipSize.height, chipParentHeight);
 }
 
+Widget _selectedInputChip({Color checkmarkColor}) {
```

Comment:
Nit spaces next to curly braces (here and on the function below).

```suggestion
Widget _selectedInputChip({ Color checkmarkColor }) {
```

Reply:
Done